alarms/dojot.auth.AuthenticationError.xml

```diff
@@ -0,0 +1,12 @@
+<alarm-metamodel>
+       <domain>AuthenticationError</domain>
+       <namespace>dojot.auth</namespace>
+       <primary-subject>
+               <primary-field type="alphanumeric">module_name</primary-field>
+               <primary-field type="alphanumeric">instance_id</primary-field>
+       </primary-subject>
+       <additional-data>
```

Comment:
perhaps here we could include more useful data to an admin observing the event, like
- userid 
- username
- reason (unknown user, incorrect password, DoS prevention)

Reply:
Ok. Will this alarm be read just by the internal components? if not, would be ok to put this details in the reason field?

Reply:
I'm not sure if I follow. What reason field? (the new one?)

If it is the new one, yes, I think it should be safe to let a user with this alarm's read permission to see this kind of information. (The user must be properly logged into the system, and have the right set of permissions to be able to see it)